pusher-rest-java.iml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't *think* you're supposed to commit this, it should be generated from the maven pom.